### DISCUSSION TITLE
Cubbit S3 - Mountain Duck says Sync is Paused

### DISCUSSION THREAD

**Discussion by luthian** [iterate-ch/cyberduck#16159] [Q&A]
Hi,

I'm using Mountain Duck Version 4.16.1 (22256) on MacOS 10.14.6 Mojave to connect to Cubbit S3. This has worked fine in the past but now I'm seeing that MD always says Sync Paused and the Cubbit S3 dot is yellow. If I click Resume Sync, it tells me it has several MB to upload and then instantly goes back to Sync Paused.

Are there logs I can pull to see if this is Cubbit's fault?

**Comment by dkocher** [comment]
Another issue with Cubbit was discussed in #16083 that has since been resolved.

**Reply by luthian** [reply]
I saw that but it doesn't seem to bear on this problem; Mountain Duck isn't showing any kind of error.

Note that I'm able to use Arq to backup to Cubbit S3 without issue.

**Comment by tatze96** [comment]
There is currently a file upload issue present in combination with Cubbit S3 that may also cause the paused synchronization on your side.

If the paused state is still present, [enable debug logging](https://docs.cyberduck.io/mountainduck/support/#debug-log), recreate the issue and send us the log files with the ticket reference GH-16159 to support@mountainduck.io for further investigation.

**Reply by luthian** [reply]
Done!

**Reply by dkocher** [reply]
Updating to version [4.16.2](https://mountainduck.io/changelog/) should resolve this interoperability issue caused by not well formed replies to `GetObjectAcl`.
